sopel/__init__.py

```diff
@@ -34,7 +34,7 @@
 ]
 
 loc = locale.getlocale()
-if not loc[1] or 'UTF-8' not in loc[1]:
+if not loc[1] or ('UTF-8' not in loc[1] and 'utf8' not in loc[1]):
```

Comment:
Shouldn't this be or not and?

Reply:
No, because in an OR, one of them would always evaluate to true, because only one of the two conditions can be true at the same time.